go.mod

```diff
@@ -29,6 +29,27 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
```

Comment:
Did you run `go mod tidy --compat=1.17`? Weird, these should be bundled with the existing `indirect` require block below.

Reply:
Yeah, definitely looks like something went not-quite-right here

Reply:
Had to manually move them to the same block and run `go mod tidy`